### PR TITLE
Adds 5 new IOK rules

### PR DESCRIPTION
**Pull request by IlluminatiFish**
- Adds 4 new IOK rules for the following imitated brands:
  - TransferWise
  - Facebook 🇵🇱 
  - Microsoft Tech Support 🇯🇵 
  - Adobe
- Additionally adds an IOK rule to detect a new anti-analysis technique using both URL & Base64 encoding
- Also fixes the description and title of the `microsoft-tech-support-0589be7` rule

### CODE REVIEW

File: indicators/adobe-5c70696.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+title: Adobe Phishing Kit 5c70696
```
Comment (bradleyjkemp) [inline indicators/adobe-5c70696.yml:1]: Looks good 👍🏻

File: indicators/base64-url-encoded-body.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+title: Base64 & URL-encoded document body
+description: |
+  To evade static analysis, the document body can be wrapped in several
+  JavaScript functions such as `decodeURIComponent` and `atob` in order
+  to evade analysis.
+  This helps defeat simple scanners which don't evaluate JavaScript.
+
```
Comment (bradleyjkemp) [inline indicators/base64-url-encoded-body.yml:7]:
```suggestion
related:
  - hex-encoded-body
```

File: indicators/transferwise-d777126.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+title: Transferwise Phishing Kit d777126
```
Comment (bradleyjkemp) [inline indicators/transferwise-d777126.yml:1]:
```suggestion
title: Wise Phishing Kit d777126

```

File: indicators/transferwise-d777126.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+title: Transferwise Phishing Kit d777126
+description: |
+    Transferwise phishing kit which uses the same sentry.io
```
Comment (bradleyjkemp) [inline indicators/transferwise-d777126.yml:3]:
```suggestion
    Wise phishing kit which uses the same sentry.io

```

File: indicators/transferwise-d777126.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+detection:
+
+    sentryAPICall:
+      requests|contains: '?sentry_key=d777126e0316438b8a582a674f5ec311&sentry_version=7'
+   
+    condition: sentryAPICall
+
+tags:
+  - target.transferwise
```
Comment (bradleyjkemp) [inline indicators/transferwise-d777126.yml:20]:
```suggestion
  - target.wise

```

File: indicators/transferwise-d777126.yml [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+references:
+    - https://urlscan.io/result/6d6dc13f-671c-4c1b-a58c-cc9470bcef2d
+    - https://urlscan.io/result/c6a7e6a2-0b60-46ea-9741-ce53d816d729
+    - https://urlscan.io/result/86f8b7e2-1484-407c-a7b7-93a497014505
+    - https://urlscan.io/result/c2970a14-2b7a-449c-ade9-e4f6fa5c414d
+
+detection:
+
+    sentryAPICall:
+      requests|contains: '?sentry_key=d777126e0316438b8a582a674f5ec311&sentry_version=7'
```
Comment (bradleyjkemp) [inline indicators/transferwise-d777126.yml:15]: I suspect this is Wise's actual Sentry key but it does only seem to fire on phishing sits now 🤔

**Review by bradleyjkemp — COMMENTED** [review]
Looks good, just had a couple of comments on the Wise rule

File: indicators/base64-url-encoded-body.yml (hunk 2 above)
Comment (IlluminatiFish) [reply indicators/base64-url-encoded-body.yml:7]: I'll keep this noted for future reference, thanks!

File: indicators/transferwise-d777126.yml (hunk 6 above)
Comment (IlluminatiFish) [reply indicators/transferwise-d777126.yml:15]: Quite peculiar, I must say haha.

Comment (IlluminatiFish) [reply indicators/transferwise-d777126.yml:17]: It must be from the old version of Wise's login page, due to there being no reference to Sentry what so ever in the Wise login page now

**Pull request merged**